src/main/resources/www.liquibase.org/xml/ns/mongodb/liquibase-mongodb-latest.xsd

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+            targetNamespace="http://www.liquibase.org/xml/ns/mongodb"
+            xmlns="http://www.liquibase.org/xml/ns/mongodb"
+            elementFormDefault="qualified">
+
+    <xsd:element name="insertOne">
+
+        <xsd:complexType>
+
+            <xsd:all>
+                <xsd:element name="document" type="xsd:string" minOccurs="1" maxOccurs="1"/>
+                <xsd:element name="options" type="xsd:string" minOccurs="0" maxOccurs="1"/>
+            </xsd:all>
+
+            <xsd:attribute name="collectionName" type="xsd:string" use="required"/>
+
+        </xsd:complexType>
+
+    </xsd:element>
+
+    <xsd:element name="insertMany">
+
+        <xsd:complexType>
+
+            <xsd:all>
+                <xsd:element name="documents" type="xsd:string" minOccurs="1" maxOccurs="1"/>
+                <xsd:element name="options" type="xsd:string" minOccurs="0" maxOccurs="1"/>
+            </xsd:all>
+
+            <xsd:attribute name="collectionName" type="xsd:string" use="required"/>
+
+        </xsd:complexType>
+
+    </xsd:element>
+
+    <xsd:element name="createCollection">
+
+        <xsd:complexType>
+
+            <xsd:sequence>
+                <xsd:element name="options" type="xsd:string" minOccurs="0" maxOccurs="1"/>
+            </xsd:sequence>
+
+            <xsd:attribute name="collectionName" type="xsd:string" use="required"/>
+
+        </xsd:complexType>
+
+    </xsd:element>
+
+    <xsd:element name="dropCollection">
+
+        <xsd:complexType>
+
+            <xsd:attribute name="collectionName" type="xsd:string" use="required"/>
+
+        </xsd:complexType>
+
+    </xsd:element>
+
+    <xsd:element name="createIndex">
+
+        <xsd:complexType>
+
+            <xsd:sequence>
+                <xsd:element name="keys" type="xsd:string" minOccurs="1" maxOccurs="1"/>
+                <xsd:element name="options" type="xsd:string" minOccurs="0" maxOccurs="1"/>
+            </xsd:sequence>
+
+            <xsd:attribute name="collectionName" type="xsd:string" use="required"/>
+
+        </xsd:complexType>
+
+    </xsd:element>
+
+    <xsd:element name="dropIndex">
+
+        <xsd:complexType>
+
+            <xsd:sequence>
+                <xsd:element name="keys" type="xsd:string" minOccurs="1" maxOccurs="1"/>
+                <xsd:element name="options" type="xsd:string" minOccurs="0" maxOccurs="1"/>
```

Comment:
This additional line is only one difference with the initial schema `dbchangelog-ext.xsd`. It was added not to fail [here](https://github.com/liquibase/liquibase-mongodb/blob/c898566138fcf8163e1a1298ef153905e76f10cb/src/test/resources/liquibase/harness/compatibility/foundational/changelogs/nosql/dropIndex.xml#L80L82)